bedrock/mozorg/templates/mozorg/about/policy/transparency/jan-jun-2020.html

```diff
@@ -208,14 +208,16 @@ <h3>Personal Data Requests</h3>
       <tbody>
         <tr>
           <th scope="row">Mozilla</th>
-          <td>320</td>
+          <td>* 601</td>
         </tr>
         <tr>
           <th scope="row">Pocket</th>
-          <td>738</td>
+          <td>* 1052</td>
         </tr>
       </tbody>
     </table>
+
+    <p>* This data was corrected on 17 February 2021.</p>
```

Comment:
NIT: I recommend setting this text in italics. 
```suggestion
    <p><em>* This data was corrected on 17 February 2021.</em></p>
```